intellij-plugin-structure/structure-teamcity-actions/src/main/kotlin/com/jetbrains/plugin/structure/teamcity/action/Validator.kt

```diff
@@ -27,14 +27,27 @@ internal fun validateTeamCityAction(descriptor: TeamCityActionDescriptor) = sequ
   validateSpecVersion(descriptor.specVersion, ActionSpecVersion.NAME, ActionSpecVersion.DESCRIPTION)
 
   validateExists(descriptor.name, ActionName.NAME, ActionName.DESCRIPTION)
-  validateNotEmptyIfExists(descriptor.name, ActionName.NAME, ActionName.DESCRIPTION)
-  validateMaxLength(descriptor.name, ActionName.NAME, ActionName.DESCRIPTION, ActionName.MAX_LENGTH)
+
+  validateExists(descriptor.getNamespace(), ActionName.Namespace.NAME, ActionName.Namespace.DESCRIPTION)
```

Comment:
It might be better to check if the composite name is in the correct format and only check the two parts if it is, otherwise names like `aaa`, `a/b/c` etc. would produce a bunch of redundant and confusing validation problems. Maybe checking the format with a regex that contains two groups for the two parts would be optimal.